src/main/java/mate/project/onlinebookshop/mapper/BookMapperImpl.java

```diff
@@ -0,0 +1,31 @@
+package mate.project.onlinebookshop.mapper;
+
+import mate.project.onlinebookshop.dto.BookDto;
+import mate.project.onlinebookshop.dto.CreateBookRequestDto;
+import mate.project.onlinebookshop.model.Book;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BookMapperImpl implements BookMapper {

```

Comment:
good job, but would be good to get experience working with Mapstruct and boost your CV :)